lib/jira/core.rb

```diff
@@ -61,17 +60,10 @@ def ticket?(ticket, verbose=true)
       ### Relevant Paths
 
       #
-      # @return [String] path to .jira-url file
+      # @return [String] path to .jira-cli file
       #
-      def url_path
-        @url_path ||= self.root_path + "/.jira-url"
-      end
-
-      #
-      # @return [String] path to .jira-auth file
-      #
-      def auth_path
-        @auth_path ||= self.root_path + "/.jira-auth"
+      def cli_path
+        @cli_path ||= self.root_path + "/.jira-cli"
```

Comment:
Not sure if I entirely agree with this. It's a better solution than what's currently in place though, so we can leave it as is for this body of work